.npmrc

```diff
@@ -1,5 +1,4 @@
 save-exact = true
 engine-strict = true
 unsafe-perm = true
-legacy-peer-deps = true
-frozen-lockfile = false
```

Comment:
`false` is already the default anyways, and for CI environment it makes sense to be true instead: https://pnpm.io/cli/install#--frozen-lockfile